.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -47,7 +45,8 @@ jobs:
           fi
         else
           if [[ -z "${KUBERNETES_VERSION}" ]]; then
-            KUBERNETES_VERSION=$(az aks get-versions -l $AZURE_BUILD_LOCATION --query "orchestrators[?default==\`true\`].orchestratorVersion" -otsv)
+            defaultMajorVersion=$(az aks get-versions -l $AZURE_BUILD_LOCATION --query "values[?isDefault==\`true\`].version" -otsv)
+            KUBERNETES_VERSION=$(az aks get-versions -l eastus -o table | awk '{print $1}' | awk "/$defaultMajorVersion/ {print}" | sort -r | awk 'NR==1')
```

Comment:
`$AZURE_BUILD_LOCATION` is used in previous line but `eastus` is used here.

Reply:
Thanks for reminding. Should use `$AZURE_BUILD_LOCATION` instead of `eastus`.